clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -135,6 +135,144 @@
                     "description": "Default list of USS paths to search for copybooks\nZowe Explorer version 1.15.0 or higher is required to download copybooks from the mainframe",
                     "uniqueItems": true
                 },
+                "cobol-lsp.cpy-manager.uss-copybook-encoding": {
```

Comment:
@zeb 
![image](https://user-images.githubusercontent.com/69206564/152154943-ac934173-6260-4ff5-b21d-a62da1dca779.png)

Reply:
Is it make sense to have a default value here?

Reply:
I see, this is related to `downloadBinary` flag.

Reply:
Yes, if nothing is provided, we download in text mode. In case encoding is provided we download in binary and make the conversion at our end.